src/allencell_ml_segmenter/main/i_viewer.py

```diff
@@ -72,3 +72,17 @@ def subscribe_layers_change_event(
         self, function: Callable[[NapariEvent], None]
     ) -> None:
         pass
+
+    @abstractmethod
+    def get_seg_layers(self, layer_list: list[Layer]) -> list[Layer]:
```

Comment:
The parameters for this and the function below are not self-explanatory to me--a comment would be useful